### DISCUSSION TITLE
llvm main

### DISCUSSION THREAD

**Discussion by dnsampaio** [pocl/pocl#1686] [General]
Hi,
here at Kalray we ported pocl to our mppa architecture. However we're moving our llvm development process to be as close as possible to upstream. So To avoid redoing work every time a new release comes I would like to upstream as soon as possible changes that allow to compile against llvm/main.

**Comment by dnsampaio** [comment]
I've sent a first patch here:
https://github.com/pocl/pocl/pull/1685

**Reply by pjaaskel** [reply]
Thanks. Good to hear that Kalray is still benefitting from PoCL and also now contributing code back!

**Reply by dnsampaio** [reply]
Yes, pocl is crucial in our software stack, and I would like to upstream as much as possible to help this very nice project.
